### ISSUE TITLE
Setup Netlify and Fix Readme

### ISSUE THREAD

**Issue by malgamves** [devconzm/devconzm-blog#1]
Need to setup repo

**Comment by OlivierJM** [comment]
anyway I can help with this?

**Comment by malgamves** [comment]
Yes please @OlivierJM, could you setup the netlify connection and sub domain? Any design suggestions?

**Comment by OlivierJM** [comment]
> Yes please @OlivierJM, could you setup the netlify connection and sub domain? Any design suggestions?

Not so sure on this one? I think on the subdomain we can consult the current hosting provider, for a netlify test account it would be easy to test it but we would have something like devconblog.netlify.com then maybe later we can link it with the site. 
For the design, Not so much but I think a basic design where we have a list of articles title with a short summary and author on the home page and when clicked it shows the full story.

**Comment by malgamves** [comment]
Yes please @OlivierJM, could you setup the netlify connection and sub domain? Any design suggestions?

**Comment by malgamves** [comment]
Hey @OlivierJM,  I did the Netlify setup, could you add a license and contributing.md file? I should open up a few issues and once the subdomain goes live. We share and start getting contributions.
